drivers/generic/generic.go

```diff
@@ -14,14 +15,13 @@ import (
 	"github.com/docker/machine/libmachine/state"
 )
 
-type Driver struct {
+type driver struct {
 	*drivers.BaseDriver
 	SSHKey string
```

Comment:
This tripped me up - in #1220 I had renamed the references to this to `SSHKeyPath`, which was wrong, because what _this_ `SSHKey` is is actually the _original_ SSH key, which is always imported into `SSHKeyPath`.

Because of this confusion, I think it would be a good idea to rename this to `SourceSSHKey` or `OrigSSHKey` or something like that. It doesn't even need to be exposed, so maybe something like `sourceSSHKey` would work.

Obviously not technically necessary, but I think a rename would make the code a lot more clear :)